compiler/src/formatting/format.re

```diff
@@ -1744,25 +1759,42 @@ and print_application =
 
   switch (expressions) {
   | [first, second] when infixop(function_name) =>
-    print_infix_application(~expressions, ~original_source, ~comments, func)
+    print_infix_application(
+      ~in_conditional?,
+      ~expressions,
+      ~original_source,
+      ~comments,
+      func,
+    )
   | _ =>
-    print_other_application(~expressions, ~original_source, ~comments, func)
+    print_other_application(
+      ~in_conditional?,
+      ~expressions,
+      ~original_source,
+      ~comments,
+      func,
+    )
   };
 }
 
 and print_infix_application =
     (
+      ~in_conditional: option(bool)=?,
```

Comment:
Small nit, but get we make all of these `in_conditional` flags to not be optional? If we are going to have a boolean flag like this hanging around (you know I don't like them in general), then I'd prefer we are required to specify "yes" or "no" so we don't miss it somewhere.